bundles/org.openhab.core.io.rest.publisher/src/main/java/org/openhab/core/thirdparty/com/eclipsesource/jaxrs/publisher/ApplicationConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2015 EclipseSource and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Holger Staudacher - initial API and implementation, ongoing development
+ ******************************************************************************/
+package org.openhab.core.thirdparty.com.eclipsesource.jaxrs.publisher;
```

Comment:
What we usually did (within bindings that required to copy 3rd party source code) is to keep the original namespace in place (i.e. `com.eclipsesource.jaxrs.publisher`) and just move those sources out from `src/main/java` to `src/3rdparty/java`. This also makes sure that an automatic license header update can nicely exclude such files. Would that be an option here as well?

Reply:
I changes the original namespace to ensure it is not used anymore (e.g. the rest.auth package used a reference to a provider class).
I don't know about all the other jersey stuff if it uses optional imports etc. And I don't want to inspect them all (better drop them some time).

I also modified and removed (only a few ATM) classes. So it is not about bundling unmodified stuff.

But sure, we could use `src/3rdparty/main` if we also "know" that it does not contain the originap sources anymore.

I do not have access to a PC this weekend, can you apply the requested changes?

Reply:
Ok, let's keep the namespace and just move the classes to `src/3rdparty/main`. I'll create a PR against your branch, sure!

Reply:
Thank you.

You are absolutely riggt, already the license header makes this necessary.

Don't forget to "inform" Maven and Eclipse to add that source folder :wink:

The mobile phonr will be enough to accept your PR.